From: Didier Raboud <odyx@debian.org>
Date: Tue, 9 Aug 2016 18:11:22 +0200
Subject: Tests: Force LC_ALL=C environment variable when grepping the output
 of lpstat

Permits testing in non-"C" environments.
Origin: vendor
Patch-Name: tests-make-lpstat-call-reproducible.patch
---
 test/run-stp-tests.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/run-stp-tests.sh b/test/run-stp-tests.sh
index 88bafb1..d9b3985 100755
--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -642,6 +642,7 @@ echo "DYLD_LIBRARY_PATH=\"$DYLD_LIBRARY_PATH\"; export DYLD_LIBRARY_PATH" >>$run
 echo "LD_LIBRARY_PATH=\"$LD_LIBRARY_PATH\"; export LD_LIBRARY_PATH" >>$runcups
 echo "LD_PRELOAD=\"$LD_PRELOAD\"; export LD_PRELOAD" >>$runcups
 echo "LOCALEDIR=\"$LOCALEDIR\"; export LOCALEDIR" >>$runcups
+echo "LC_ALL=\"C\"; export LC_ALL" >>$runcups
 if test "x$CUPS_DEBUG_LEVEL" != x; then
 	echo "CUPS_DEBUG_FILTER='$CUPS_DEBUG_FILTER'; export CUPS_DEBUG_FILTER" >>$runcups
 	echo "CUPS_DEBUG_LEVEL=$CUPS_DEBUG_LEVEL; export CUPS_DEBUG_LEVEL" >>$runcups
